lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -145,7 +144,10 @@
                                 message = response.user_message;
                             }
                             view.showMessage(message, message);
-                        });
+                        })
+                    ViewUtils.disableElementWhileRunning($(".action-primary"), function() {
+                        return deferred.promise();
```

Comment:
I find this hard to read when the function in question is just a promise. I think it would be clearer if you moved the ``save`` call inside and then you don't need a variable too:

    ViewUtils.disableElementWhileRunning($(".action-primary"), function() {
        return this.teamModel.save(data, saveOptions)
            ...
    });